modules/virt-hot-unplugging-a-virtual-disk-using-the-web-console.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+[id="virt-hot-unplugging-a-virtual-disk-using-the-web-console{context}"]
+= Hot-unplugging a virtual disk using the web console
+
+Hot-unplug virtual disks that you want to attach to a virtual machine instance (VMI) while a virtual machine is running.
+
+.Prerequisites
+* Your virtual machine must be running with a hot-plugged disk attached.
+
+.Procedure
+
+. Click *Workloads → Virtualization → Virtual Machines* from the side menu.
+
+. Select the running virtual machine with the disk you want to hot-unplug.
+
+. In the *Disks* tab, click the kebab menu of the hot-plugged disk you want to hot-unplug and click *Delete*.
```

Comment:
Per ISG, do not use "kebab menu". Instead, we refer to this as the *Options* menu followed by the icon image. 

. Click the Options menu {kebab} of the virtual disk that you want to hot-unplug, and then click *Delete*.